apps/utils/draft_pull_request.py

```diff
@@ -37,10 +93,27 @@
     required=False,
 )
 @click.option(
-    "--base-branch", type=str, default=None, help="Name of base branch (if not given, 'master' will be used)."
+    "--base-branch", "-b", type=str, default=None, help="Name of base branch (if not given, 'master' will be used)."
+)
+@click.option("--title", "-t", type=str, default=None, help="Title of the PR and the first commit.")
+@click.option(
+    "--category",
+    "-c",
+    type=click.Choice(list(PR_CATEGORIES.keys()), case_sensitive=False),
+    help=f"Category of the PR. Only works if --title is used (it will be added to the PR title). A corresponding emoji will be pre-fixed to the PR title.\n {description}",
```

Comment:
Maybe rephrase it a bit: "Category of the PR (only relevant if --title is given). A corresponding emoji will be prepended to the title.\n {description}"